hammer_cli_foreman_discovery.gemspec

```diff
@@ -21,6 +21,7 @@ DESC
   s.extra_rdoc_files = Dir['LICENSE', 'README*']
   s.require_paths = ["lib"]
 
-  s.add_dependency "hammer_cli_foreman", ">= 0.1.2"
-  s.add_development_dependency 'rake'
+  s.add_dependency 'hammer_cli_foreman', '~> 3.9'
```

Comment:
`~> 3`

Reply:
I first thought that too in https://github.com/theforeman/hammer-cli-foreman-leapp/pull/7#discussion_r1498069204 but this will also allow 3.10 and 3.11 so this is fine. `~> 3` will also allow 4.x so I think `~> 3.9` is better.